README.md

```diff
@@ -1,24 +1,66 @@
-# basic-storage-integration
+<p>
+<a href="#start"><img height="30rem" src="https://raw.githubusercontent.com/arcana-network/branding/main/an_logo_light_temp.png"/></a>
+</p>
+<p>
+<a title="MIT License" href="https://github.com/arcana-network/license/blob/main/LICENSE.md"><img src="https://img.shields.io/badge/license-MIT-blue"/></a>
+<a title="Beta release" href="https://github.com/arcana-network/basic-storage-wallet-integration/releases"><img src="https://img.shields.io/github/v/release/arcana-network/basic-storage-wallet-integration?style=flat-square&color=28A745"/></a>
+<a title="Twitter" href="https://twitter.com/ArcanaNetwork"><img alt="Twitter URL" src="https://img.shields.io/twitter/url?style=social&url=https%3A%2F%2Ftwitter.com%2FArcanaNetwork"/></a>
+</p><p id="start" align="center">
+</p>
+
+# Sample Code: Basic Storage and Auth SDK integration
+
+The repository `basic-storage-wallet-integration` contains sample code to demonstrate how you can integrate with Arcana [Auth](https://github.com/arcana-network/auth) and [Storage](https://github.com/arcana-network/storage) SDKs.
+
+It does not have any code for invoking other features and functions of these integrated SDKs.  Check out Arcana [Sample Demo App](https://github.com/arcana-network/demo-app) repository for a full fledged sample dApp that integrates with Arcana SDKs and plugs in Auth and Storage functionalities.
+
+## ⚙️ Installation
+
+### Deploy Software Dependencies
```

Comment:
Deploy? Shouldn't it be something like “Download”?

Reply:
```suggestion
### Download Software Dependencies